sys-process/bottom/bottom-0.6.3.ebuild

```diff
@@ -0,0 +1,199 @@
+# Copyright 2020-2021 Gentoo Authors
```

Comment:
did that ebuild exist in 2020? if not, should be just 20201

Reply:
This ebuild (or rather, an ancestor of this file) is currently part of `::guru`, and @juippis had written to suggest moving it to `::gentoo`. It was originally written in 2020:

https://cgit.gentoo.org/repo/proj/guru.git/commit/sys-process/bottom?id=615d58ffcdd2233e4e5a3baaafea35d32afa5d61